### PR TITLE
DATAUP-305 work on configure tab

### PR DESCRIPTION
**Pull request by briehl**
# Description of PR purpose/changes

Some initial work on the configure tab. This renders a basic configure tab without populating parameters or files.
There's also some modification here to the bulk import cell to start up with it activated (until we have state saving worked out).

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X
- [x] Added the Jira Ticket to the title of the PR e.g. (DATAUP-69 Adds a PR template)

# Testing Instructions
* Details for how to test the PR: 
- [ ] Tests pass in travis and locally 
- [ ] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [ ] My code follows the guidelines at https://sites.google.com/truss.works/kbasetruss/development
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: nbextensions/bulkImportCell/bulkImportCell.js [hunk 1]
```diff
@@ -28,18 +28,24 @@ define([
 
     const div = html.tag('div');
 
-    class DefaultWidget {
-        constructor() {
-
-        }
+    function DefaultWidget() {
+        function make() {
+            function start(options) {
```
Comment (eamahanna) [inline nbextensions/bulkImportCell/bulkImportCell.js:33]: Should we remove options until it is used?

File: nbextensions/bulkImportCell/bulkImportCell.js [hunk 2]
```diff
@@ -223,7 +230,7 @@ define([
             if (this.tabWidget !== null) {
                 this.tabWidget.stop();
             }
-            this.tabWidget = new this.tabSet.tabs[tab].widget();
+            this.tabWidget = this.tabSet.tabs[tab].widget.make({bus: this.bus});
```
Comment (eamahanna) [inline nbextensions/bulkImportCell/bulkImportCell.js:233]: Does every app widget require a bus?

File: nbextensions/bulkImportCell/tabs/configure.js [hunk 3]
```diff
@@ -1,24 +1,97 @@
...
-            this.container.innerHTML = 'configure some things!';
+        function renderLayout() {
+            let events = Events.make(),
+                formContent = [
+                    ui.buildPanel({
+                        title: span([
+                            'File Paths',
+                            span({
+                                dataElement: 'advanced-hidden-message',
+                                style: {
```
Comment (eamahanna) [inline nbextensions/bulkImportCell/tabs/configure.js:50]: Do we want to remove the inline styling?

File: test/unit/karma.conf.js [hunk 4]
```diff
@@ -33,7 +33,7 @@ module.exports = function (config) {
         },
         files: [
             'kbase-extension/static/narrative_paths.js',
-            {pattern: 'test/unit/spec/**/*.js', included: false},
+            {pattern: 'test/unit/spec/nbextensions/**/*.js', included: false},
```
Comment (eamahanna) [inline test/unit/karma.conf.js:36]: Is this suppose to be checked in?

File: nbextensions/bulkImportCell/bulkImportCell.js (hunk 1 above)
Comment (briehl) [reply nbextensions/bulkImportCell/bulkImportCell.js:33]: Really that whole `DefaultWidget` is just there as a placeholder. I'll remove the options bit.

File: nbextensions/bulkImportCell/bulkImportCell.js (hunk 2 above)
Comment (briehl) [reply nbextensions/bulkImportCell/bulkImportCell.js:233]: Yes, eventually. Well, possibly not, but this allows consistency in widget building.

Comment (briehl) [reply nbextensions/bulkImportCell/bulkImportCell.js:233]: I mean, the info tab probably won't need one.

File: nbextensions/bulkImportCell/tabs/configure.js (hunk 3 above)
Comment (briehl) [reply nbextensions/bulkImportCell/tabs/configure.js:50]: Yup, I can do that.

File: test/unit/karma.conf.js (hunk 4 above)
Comment (briehl) [reply test/unit/karma.conf.js:36]: I coulda sworn I fixed that... Thanks for catching!

**Review by eamahanna — APPROVED** [review]

**Pull request merged**